plugins/com.google.cloud.tools.eclipse.projectselector.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Tests for Cloud Tools for Eclipse Project Selector Widget
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.projectselector.test
+Bundle-Vendor: Google Inc.
+Bundle-Version: 0.1.0.qualifier
+Fragment-Host: com.google.cloud.tools.eclipse.projectselector
+Require-Bundle: com.google.guava;bundle-version="[20.0.0,21.0.0)",
+ org.hamcrest;bundle-version="1.1.0",
+ org.junit;bundle-version="4.12.0"
+Import-Package: org.mockito;provider=google;version="1.10.19",
+ org.mockito.runners;provider=google;version="1.10.19",
+ org.mockito.stubbing;provider=google;version="1.10.19",
+ org.objenesis;provider=google;version="2.2.0"
```

Comment:
I believe this is the magical combination we need to run mockito tests without classloading issues.